### PR TITLE
intro slides

### PR DESCRIPTION
**Pull request by kkmann**
#4 

- [ ] Friedrich bio
- [ ] Daniel bio

### CODE REVIEW

File: slides/01_intro.qmd [hunk 1]
```diff
@@ -26,120 +22,100 @@ employer or company.*
...
-1. Encapsulate behavior (functions)
-2. Avoid global state/variables
-3. Adopt consistent coding style
-4. Document well
-5. Version your code
-6. Share as 'bundle'
+1.  Encapsulate behavior (functions)
+2.  Avoid global state/variables
+3.  Adopt consistent coding style
+4.  Document well
```
Comment (danielinteractive) [inline slides/01_intro.qmd:138]: probably "Add tests" would be good to add?

**Review by danielinteractive — APPROVED** [review]
thank you Kevin, looks great! only one minor comment :-)

**Comment by kkmann** [comment]
@danielinteractive @fpahlke happy for you to add bio data in this PR

**Review by fpahlke — APPROVED** [review]
Great, Kevin :)
Friedrich's bio data added; minor improvements

**Pull request merged**